### PR TITLE
Add dropdown to select db for raw_couch

### PR DESCRIPTION
**Pull request by esoergel**
@sravfeyn

### CODE REVIEW

**Comment by esoergel** [comment]
@dimagi/team-commcare-hq Minor update to [raw couch](https://www.commcarehq.org/hq/admin/raw_couch/) - now you can search only a specific database (this is something I've wanted a couple times recently when comparing documents migrated from the main db to a new one).

**Comment by dannyroberts** [comment]
cool

File: corehq/apps/hqadmin/templates/hqadmin/raw_couch.html [hunk 1]
```diff
@@ -7,10 +7,13 @@
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'style/lib/bootstrap-3.2.0/dist/css/bootstrap.min.css' %}"/>
         <link rel="stylesheet" href="{% static 'codemirror/lib/codemirror.css' %}" />
         <link rel="stylesheet" href="{% static 'codemirror/addon/fold/foldgutter.css' %}"/>
+        <link type="text/css" rel="stylesheet" media="all" href="{% new_static 'style/lib/select2/select2.css' %}" />
```
Comment (benrudolph) [inline corehq/apps/hqadmin/templates/hqadmin/raw_couch.html]: this should pull from bower

Comment (dannyroberts) [inline corehq/apps/hqadmin/templates/hqadmin/raw_couch.html]: good point. Looks like this version is 3.5.2, the version we have in bower is 3.4.5 ("select2-3.4.5-legacy"), and also in /style/ we have a 4.0...,

Comment (esoergel) [inline corehq/apps/hqadmin/templates/hqadmin/raw_couch.html]: I have an even better solution! https://github.com/dimagi/commcare-hq/pull/9397

**Pull request merged**